websocket-codec/src/message.rs

```diff
@@ -275,21 +277,21 @@ mod tests {
         header.write_to(&mut bytes);
 
         {
-            let data = data.as_bytes().into();
+            let mut vec = Vec::with_capacity(data.len());
+            vec.extend_from_slice(data.as_bytes());
+            let borrowed_data = vec.into();
             let data = if let Some(mask) = header.mask {
-                Masker::new().mask(data, mask)
+                Masker::new().mask(borrowed_data, mask)
             } else {
-                data
+                borrowed_data
             };
-
             bytes.put(data);
         }
 
         let message2 = decode(&mut MessageCodec::new(), &bytes).unwrap();
         assert_eq!(is_text, message2.as_text().is_some());
         assert_eq!(data.as_bytes(), message2.data());
     }
-
```

Comment:
Can you check that you ran `cargo fmt`?

Reply:
it's automatically launched on save in vim but will check